json-schemas/workflowRunsRequest.json

```diff
@@ -4,6 +4,9 @@
         "todoRemove": {
             "type": "object",
             "properties": {
+              "authenticityToken": {
```

Comment:
super nit - do you mind moving this one to the end? XD sorry

Reply:
can do.  I thought they were alphabetical, but not quite